src/astar/common.js

```diff
@@ -80,8 +84,7 @@ export const scheduleTask = async ({
     console.log(`\n2. One-time proxy setup on ${oakChainName} ...`);
     console.log(`\na) Add a proxy for ${keyringPair.meta.name} If there is none setup on ${oakChainName} (paraId:${astarChainData.paraId})\n`);
     const proxyTypeOak = 'Any';
-    console.log('astarChainData.relayChain: ', astarChainData.relayChain);
-    const proxyAccoundIdOnOak = oakAdapter.getDerivativeAccount(u8aToHex(keyringPair.addressRaw), astarChainData.paraId, { network: astarChainData.network, locationType: 'XcmV3MultiLocation' });
+    const proxyAccoundIdOnOak = oakAdapter.getDerivativeAccount(u8aToHex(keyringPair.addressRaw), astarChainData.paraId, { network: astarChainData.xcm.network, locationType: 'XcmV3MultiLocation' });
```

Comment:
it needs to be `astarChainData.xcm.network` since we changed the config library.